databases/foundationdb/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+name                foundationdb
+version             6.3.12
+categories          databases
+license             Apache-2
+maintainers         {me.com:spam_brian @brianwells} openmaintainer
+platforms           darwin
+description         FoundationDB is a distributed database providing ACID transactions.
+long_description    \
+    FoundationDB is a scalable, fault-tolerant, ordered \
+    key-value store with full ACID transactions.  This port \
+    will install version ${version} of the FoundationDB \
+    binaries suitable for development on macOS.
+
+homepage            https://www.foundationdb.org/
+
+github.setup        apple foundationdb ${version}
+
+checksums           rmd160  2bf9a258d5db140d232fb6355fc810010824c014 \
+                    sha256  034f69d4ddf7faccd7cae2a6438d64643ac98a3a9c7b3e6ed2a03105c480df95 \
+                    size    9694306
+
+depends_lib-append  port:mono
+
+cmake.generator     Ninja
+
+# The FoundationDB build can easily run out of memory and crash, 
+# so the recommendation is to use "ninja -j1" when building.
+build.jobs          1
+
+destroot {
+    # install compiled binaries
+
+    xinstall -d -m 755 ${destroot}${prefix}/bin
+    xinstall -m 755 ${workpath}/build/packages/bin/fdbcli \
+        ${workpath}/build/packages/bin/fdbbackup \
+        ${destroot}${prefix}/bin
+    foreach x {backup_agent dr_agent fastrestore_tool fdbdr fdbrestore} {
+        ln -f -s fdbbackup ${destroot}${prefix}/bin/${x}
```

Comment:
Is this line correct ? To my eye it looks like it is generating links back to the destroot area, which won’t exist after installation ?

Reply:
The links point locally to `fdbbackup` and are placed in `/opt/local/bin` upon install:
```
lrwxr-xr-x  1 root  admin  9 Apr 24 09:27 backup_agent -> fdbbackup
lrwxr-xr-x  1 root  admin  9 Apr 24 09:27 dr_agent -> fdbbackup
lrwxr-xr-x  1 root  admin  9 Apr 24 09:27 fastrestore_tool -> fdbbackup
lrwxr-xr-x  1 root  admin  9 Apr 24 09:27 fdbdr -> fdbbackup
lrwxr-xr-x  1 root  admin  9 Apr 24 09:27 fdbrestore -> fdbbackup
```

Reply:
yes, of course, my mistake...